pkg/minikube/cruntime/cri.go

```diff
@@ -134,10 +134,9 @@ func pauseCRIContainers(cr CommandRunner, root string, ids []string) error {
 		args = append(args, "--root", root)
 	}
 	args = append(args, "pause")
-	cargs := args
 	for _, id := range ids {
-		cargs = append(cargs, id)
-		if _, err := cr.RunCmd(exec.Command("sudo", cargs...)); err != nil {
+		args := append(args, id)
```

Comment:
you still need a local variable, no ? maybe call it something different than the global one, less confusion ?

Reply:
if I do something like `cargs := append(args, id)` then golint complains

Reply:
```
pkg/minikube/cruntime/cri.go:138:12: appendAssign: append result not assigned to the same slice (gocritic)
		cargs := append(args, id)
```

Reply:
How about keeping the cargs, I am not confident what would be side effects of getting rid of that local cargs? Would at some point the inner args append stuff to the outer args ?

Reply:
the local cargs is what's causing the bug. i'm redefining args inside of the loop, so tainting the outer args isn't going to happen.

Reply:
gotcha